ngraph/python/CMakeLists.txt

```diff
@@ -16,30 +16,71 @@
 
 cmake_minimum_required (VERSION 3.13)
 
-project (pyngraph)
-
-include(ExternalProject)
-
-ExternalProject_Add(
-        pybind11
-        GIT_REPOSITORY "https://github.com/pybind/pybind11.git"
-        GIT_TAG "v2.5.0"
-        SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}/pybind11"
-        CONFIGURE_COMMAND ""
-        BUILD_COMMAND ""
-        INSTALL_COMMAND ""
-)
+project (_pyngraph)
```

Comment:
Why did you add `_` prefix?

Reply:
https://github.com/openvinotoolkit/openvino/blob/9ec18eeb9bf8f2dd5c39a7c2acacbc8a7b0c3332/ngraph/python/setup.py#L264